docs/pages/workflow/linking.md

```diff
@@ -139,15 +139,15 @@ On iOS, `Linking.canOpenURL` requires additional configuration to query other ap
   }
 ```
 
-If you don't specify this list, `Linking.canOpenURL` may return `false` regardless of whether the device has the app installed. Note that this configuration can only be tested in standalone apps, because it requires native changes that will not be applied when testing in Expo client.
+If you don't specify this list, `Linking.canOpenURL` may return `false` regardless of whether the device has the app installed. Note that this configuration can only be tested in standalone apps, because it requires native changes that will not be applied when testing in Expo Go.
 
 ## Linking to your app
 
-### In the Expo client
+### In the Expo Go app
 
-Before continuing it's worth taking a moment to learn how to link to your app within the Expo client. The Expo client uses the `exp://` scheme, but if we link to `exp://` without any address afterwards, it will open the app to the main screen.
+Before continuing it's worth taking a moment to learn how to link to your app within the Expo Go app. the Expo Go app uses the `exp://` scheme, but if we link to `exp://` without any address afterwards, it will open the app to the main screen.
```

Comment:
```suggestion
Before continuing it's worth taking a moment to learn how to link to your app within the Expo Go app. Expo Go uses the `exp://` scheme, but if we link to `exp://` without any address afterwards, it will open the app to the main screen.
```